src/game.js

```diff
@@ -0,0 +1,38 @@
+class Game {
```

Comment:
Game class has simple conditionals which are nice to follow. 

I like the use of guard clauses. 

There is the potential to pull out some functionality into private methods